src/main/java/org/jabref/gui/mergeentries/MergeEntriesAction.java

```diff
@@ -57,10 +58,9 @@ public void execute() {
             // Remove the other two entries and add them to the undo stack (which is not working...)
             NamedCompound ce = new NamedCompound(Localization.lang("Merge entries"));
             ce.addEdit(new UndoableInsertEntry(basePanel.getDatabase(), mergedEntry.get()));
-            ce.addEdit(new UndoableRemoveEntry(basePanel.getDatabase(), one));
-            basePanel.getDatabase().removeEntry(one);
-            ce.addEdit(new UndoableRemoveEntry(basePanel.getDatabase(), two));
-            basePanel.getDatabase().removeEntry(two);
+            List<BibEntry> entriesToRemove = Arrays.asList(one, two);
```

Comment:
`Lists.of(one, two)` should work, too?

Reply:
I'm not sure. I just want to create a `List` of the two `BibEntry`s.